pubmedcentral/to-dolma.py

```diff
@@ -0,0 +1,74 @@
+import argparse
+import datetime
+import functools
+import os
+
+from licensed_pile.licenses import PermissiveLicenses
+from licensed_pile.write import to_dolma
+
+parser = argparse.ArgumentParser(description="Collect PubMedCentral into Dolma format.")
+parser.add_argument("--filelist", help="The path to the filelist.txt file.")
+parser.add_argument(
+    "--data_dir", default="data/raw/", help="Path to the directory of markdown files."
+)
+parser.add_argument(
+    "--output_dir",
+    default="data/dolma/",
```

Comment:
Other datasources have been using `data/${source_name}` as the output dir, we should follow this.